src/redux/books/books.js

```diff
@@ -0,0 +1,68 @@
+import axios from 'axios';
+
+const GET_BOOKS_SUCCESS = 'bookStore/books/GET_BOOKS_SUCCESS';
+const GET_BOOKS_FAILURE = 'bookStore/books/GET_BOOKS_FAILURE';
+const ADD_BOOK_SUCCESS = 'bookStore/books/ADD_BOOK_SUCCESS';
+const ADD_BOOK_FAILURE = 'bookStore/books/ADD_BOOK_FAILURE';
+const REMOVE_BOOK_SUCCESS = 'bookStore/books/REMOVE_BOOK_SUCCESS';
+const REMOVE_BOOK_FAILURE = 'bookStore/books/REMOVE_BOOK_FAILURE';
+
+const initialState = [];
+
+export const getBooks = () => (dispatch) => axios.get('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/CBviEs2A2Tucc9CFaZmG/books').then(
+  (books) => {
+    const booksArr = Object.entries(books.data);
+    const newBooksArr = [];
+    for (let i = 0; i < booksArr.length; i += 1) {
+      const newBook = {
+        id: booksArr[i][0],
+        title: booksArr[i][1][0].title,
+        category: booksArr[i][1][0].category,
+      };
+      newBooksArr.push(newBook);
+    }
+    dispatch({ type: GET_BOOKS_SUCCESS, payload: newBooksArr });
+  },
+  (err) => dispatch({ type: GET_BOOKS_FAILURE, err }),
+);
```

Comment:
- [ ] When the app loads no book it's being load from the API. Please make sure that when the app loads it gets the books from the API provided, but whenever I change from tab it does not load the books again.

> Data from the API requests should be stored correctly in Redux.
Server requests should made in the componentDidMount lifecycle method or with react hooks.